prime-dotnet-webapi/ViewModels/Enrollee/EnrolleeRemoteAccessSiteViewModel.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Newtonsoft.Json;
+using Prime.Models;
+
+namespace Prime.ViewModels
+{
+    public class EnrolleeRemoteAccessSiteViewModel
+    {
+        [Key]
```

Comment:
This is not a model in our database, so you shouldnt need a [Key] annotation or the `using System.ComponentModel.DataAnnotations`